src/dynamics_tendencies.jl

```diff
@@ -0,0 +1,216 @@
+"""
+Compute the spectral tendency of the surface pressure logarithm
+"""
+function surface_pressure_tendency!(Prog::PrognosticVariables{NF}, # Prognostic variables
+                                   Diag::PrognosticVariables{NF}, # Diagnostic variables
+                                   l2::Int,                       # leapfrog index 2 (time step used for tendencies)
+                                   C::Constants{NF}
+                                   ) where {NF<:AbstractFloat}
+
+
+    @unpack pres_surf= Prog 
+    @unpack pres_surf_tend = Diag.Tendencies
+
+    @unpack u_grid,v_grid,div_grid = Diag.gridvars
+    @unpack u_mean,v_mean,d_mean,dumc,px,py = Diag.miscvars
+    @unpack nlev, dhs = C 
```

Comment:
`nlev` is currently not propagated into constants, but we can change that. However, you can also just obtain `nlev` from any prognostic/diagnostic variable `var` via `_,_,nlev = size(var)` which is more independent of any renamings that we still might do

Reply:
Oh and neither is `dhs` (what is that?)

Reply:
`nlev` now determined via `size`.

`dhs` is the layer thickness. I guess this will be something that will ultimately calculated and held in the `Constants` struct. For now it is just a placeholder